server/dummyData/dummyAdmins.js

```diff
@@ -8,4 +8,11 @@ module.exports = [
     type: 'admin',
     employeeId: '123123',
   },
+  {
+    _id: ObjectID('519f1f77bcb16cd788439199'),
+    username: 'steven.bianchi@beamery.com',
+    password: 'admin',
+    type: 'admin',
+    employeeId: '1',
+  },
```

Comment:
dummyAdmins on the db just has the second admin object here, which I think is fine for now?

Reply:
in fact, it's preferable since the first login will fail your new Regex

Reply:
Correct! First admin login is redundant but have kept in incase tests use it.